massa-bootstrap/src/establisher.rs

```diff
@@ -77,7 +77,21 @@ pub mod types {
         /// # Argument
         /// * `addr`: `SocketAddr` we want to bind to.
         pub async fn get_listener(&mut self, addr: SocketAddr) -> io::Result<DefaultListener> {
```

Comment:
Indeed it would be better. Looking into it, not sure what would be the best way to achieve this. The two establishers are very similar but not quite, so exporting the Establisher directly does not seem possible.
And calling in the establisher a non-impl function like the following does not seem so clean...
Maybe we should keep this for a refactor. What do you think?

```rust
use massa_network_exports::get_listener_for_establisher;
impl Establisher {
    pub async fn get_listener(&mut self, addr: SocketAddr) -> io::Result<DefaultListener> {
       get_listener_for_establisher(addr)
    }
}

```

Reply:
let's keep it for later, not a big deal ^^